js/web/lib/wasm/session-options.ts

```diff
@@ -112,6 +112,19 @@ const setExecutionProviders =
                       `Can't set a session config entry: 'preferredLayout' - ${webgpuOptions.preferredLayout}.`);
                 }
               }
+              if (webgpuOptions?.graphCaptureEnabled) {
+                if (webgpuOptions.graphCaptureEnabled !== true && webgpuOptions.graphCaptureEnabled !== false) {
```

Comment:
`webgpuOptions.graphCaptureEnabled` must be a truthy value otherwise code will not go there.

If you want to strictly check the type, in the outer if, check 
```
if (webgpuOptions?.graphCaptureEnabled !== undefined && typeof webgpuOptions.graphCaptureEnabled !== 'boolean') {
  // report type error
}
```